pkg/storaged/dialog.jsx

```diff
@@ -275,7 +290,71 @@ function is_visible(field, values) {
     return !field.options || field.options.visible == undefined || field.options.visible(values);
 }
 
-const Body = ({ body, fields, values, errors, onChange }) => {
+class Revealer extends React.Component {
+    constructor() {
+        super();
+        this.state = { revealed: false };
+    }
+
+    render() {
+        return (
+            <div>
+                <a onClick={event => { if (event.button == 0) this.setState({ revealed: !this.state.revealed }); }}>
+                    {this.props.summary}
+                </a> <i className={this.state.revealed ? "fa fa-angle-down" : "fa fa-angle-right"} />
+                <br />
+                {this.state.revealed && this.props.children}
+            </div>
+        );
+    }
+}
+
+function install_data_summary(data) {
+    if (!data || data.missing_names.length == 0)
+        return null;
+
+    let summary;
+    if (data.extra_names.length == 0)
+        summary = fmt_to_fragments(_("The $0 package will be installed."), <strong>{data.missing_names[0]}</strong>);
+    else
+        summary = fmt_to_fragments(cockpit.ngettext(_("The $0 package and $1 other package will be installed."),
+                                                    _("The $0 package and $1 other packages will be installed."),
+                                                    data.extra_names.length), <strong>{data.missing_names[0]}</strong>, data.extra_names.length);
+    if (data.remove_names.length > 0) {
+        summary = <>
+            {summary}
+            <br />
```

Comment:
Why is there a `<br>`?

Either an element should be set to block or if it's text and you want paragraphs, you should use `<p>`s instead.